pipenv/utils.py

```diff
@@ -310,11 +310,13 @@ class PipCommand(basecommand.Command):
         click_echo(
             "{0}: Your dependencies could not be resolved. You likely have a "
             "mismatch in your sub-dependencies.\n  "
-            "You can use {1} to bypass this mechanism, then run {2} to inspect "
-            "the situation.\n  "
-            "Hint: try {3} if it is a pre-release dependency."
+            "First try clearing your dependency cache with {1} and trying again.\n "
```

Comment:
This sentense need to be rewritten. First try… trying?

Reply:
True, I have corrected the sentence.